kokoro/linux/build.sh

```diff
@@ -81,6 +81,9 @@ do
 done
 echo $(date): Smoketests completed.
 
+# Build the symbol files for Android
+build -c dbg //gapidapk/android/apk:armeabi-v7a_gapir.sym //gapidapk/android/apk:arm64-v8a_gapir.sym //gapidapk/android/apk:x86_gapir.sym
```

Comment:
Does it has to be `-c dbg`? This probably means you will have a lot of things to recompile in debug mode.

Reply:
When I try to build without -c, the function name was still available but the line information disappeared. Is this something we accept?

Reply:
-c dbg will also not produce the same binary at all.

Reply:
On the build systems, we build with `-config symbols`, which adds `-g`, which should be enough to get the complete debug info for symbols, is it not?